tests/codeExecutor.test.js

```diff
@@ -16,13 +21,15 @@ describe("Code executor should work properly", () => {
             expect(result.err).toBeFalsy()
             done()
         })
-    })
+    }, 10000)
```

Comment:
To też nie powinna być stała?

Reply:
Używamy tego w jednym miejscu, więc nie wiem, czy jest sens